src/components/CmsZone.vue

```diff
@@ -12,6 +12,7 @@
     </button>
 
     <cms-inspect-sheet
+      v-if="shouldShowInspectModal"
```

Comment:
This is just to avoid putting the component into the render tree at all. It makes the vue extension work a bit better.

Reply:
thanks for cleaning this up